publish.gradle

```diff
@@ -13,23 +13,44 @@
 // limitations under the License.
 
 apply plugin: 'maven-publish'
+apply plugin: 'signing'
+
 afterEvaluate {
+
     publishing {
         repositories {
             maven {
+                name = 'mavenRepo'
```

Comment:
ℹ️ I kept it as mavenRepo cause I have no idea if we want it in the future or not.
Anyhow, we can get rid of it if there are no use cases.